### ISSUE TITLE
Implementation of chocomufin software for GitHub Actions

### ISSUE THREAD

**Issue by malamatenia** [HN-2021-Boccacius/git-project-Boccace#33]
Following the practices of HTR-United as per this far, I found it interesting to implement the software ```chocomufin``` developed by Professors Clérice and Pinche in order to ensure the quality of our transcription. All relevant information can be found detailed here: https://github.com/PonteIneptique/choco-mufin but I will explain briefly the commands.

***Note: the name did not run under choco-mufin but under chocomufin without a tilde so I am following this writing***

1.The software can be installed via the terminal with the command ```pip install chocomufin``` inside the cloned repository/
Two possibilities follow for the 
2. Creation of a spacial characters table
- A table with the special characters does not exist inside the repository (```table.csv```), so we proceed to create it with the command ```chocomufin  generate table.csv nameofyourfolder/**/*.xml``` This structure concerns a folders that contain one folder ```/**/``` that contain any .XML file ```/*.xml```. A table is created and pops up in the terminal.
- Someone else has already created a ```table.csv``` and all we need is to enhance it (as owners) or propose to enhance it (as external collaborator) with the special characters contained in another respective .xml document. In this case, the cloned repository contains already a ``` table.csv``` so we proceed to convert it as follows
3. After creating a branch (as the document is going to be pushed in the repository) the command  ```chocomufin control table.csv nameoffolder/**/*/.xml reads the xml files to get the characters that are missing from the existing table.
4. Then in order to add the ones that are not included and covert the table we type ```chocomufin convert table.csv nameoffolder/**/*/.xml```
5. Lastly, we push it to the repository to have the sum of the characters in both documents (for the moment).

The workflow: with each pull and push request the software checks if the new .xml  documents are complying with the character table created. We can very well, if we find a new character, add it following the same process.

:tada:

Well, it still does not run smoothly for the moment but I am on it :+1:

**Comment by apogonoe** [comment]
Do we still try to include this in the project?

**Comment by malamatenia** [comment]
Yes, there is a small bug I am trying to figure out with professor Clérice, which should be resolved eventually. There is no pressure to have it  debugged for the homework assignment, no pressure.
